java.lang.NoClassDefFoundError: com/sun/jna/platform/win32/SspiUtil$ManagedSecBufferDesc

Issue started after merge of #1483 on `atlas-int` docker container. Need to check waffle-jna version and upgrade as necessary.

```
30-Sep-2020 13:55:03.825 SEVERE [localhost-startStop-1] org.apache.catalina.core.ContainerBase.addChildInternal ContainerBase.addChild: start: 

Caused by: org.springframework.beans.factory.BeanCreationException:
 Error creating bean with name 'shiroFilter' defined in class path resource [org/ohdsi/webapi/ShiroConfiguration.class]:
 Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: 
Failed to instantiate [org.apache.shiro.spring.web.ShiroFilterFactoryBean]:
 Factory method 'shiroFilter' threw exception; nested exception is org.springframework.beans.factory.BeanCreationException: 
Error creating bean with name 'securityManager' defined in class path resource [org/ohdsi/webapi/ShiroConfiguration.class]: 
Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: 
Failed to instantiate [org.apache.shiro.web.mgt.DefaultWebSecurityManager]: 
Factory method 'securityManager' threw exception; nested exception is java.lang.NoClassDefFoundError: 
com/sun/jna/platform/win32/SspiUtil$ManagedSecBufferDesc
```

--- a/pom.xml
+++ b/pom.xml
@@ -14,7 +14,8 @@
     <!-- Spring Boot manages spring.version as well -->
     <spring.boot.version>1.5.20.RELEASE</spring.boot.version>
     <flyway.version>4.2.0</flyway.version>
-    <waffle.version>1.7.3</waffle.version>
+    <waffle.version>2.2.1</waffle.version>
+    <jna.version>5.5.0</jna.version>
 
     <circe.version>1.9.0-SNAPSHOT</circe.version>
     <jersey.version>2.14</jersey.version>
@@ -690,9 +691,29 @@
     </dependency>
 
     <dependency>
-      <groupId>com.github.dblock.waffle</groupId>
+      <groupId>com.github.waffle</groupId>
       <artifactId>waffle-jna</artifactId>
       <version>${waffle.version}</version>
+      <exclusions>
+        <exclusion>
+          <groupId>net.java.dev.jna</groupId>
+          <artifactId>jna</artifactId>
+        </exclusion>
+        <exclusion>
+          <groupId>net.java.dev.jna</groupId>
+          <artifactId>jna-platform</artifactId>
+        </exclusion>
+      </exclusions>
+    </dependency>
+    <dependency>
+      <groupId>net.java.dev.jna</groupId>
+      <artifactId>jna</artifactId>
+      <version>${jna.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>net.java.dev.jna</groupId>
+      <artifactId>jna-platform</artifactId>
+      <version>${jna.version}</version>
     </dependency>
     <dependency>
       <groupId>io.jsonwebtoken</groupId>

